.github/pull_request_template.md

```diff
@@ -17,7 +17,11 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: `npm run dev`.
+4. Check localhost:3000 for changes.
```

Comment:
Just a minor nitpick, but consider linking the [localhost:3000](http://localhost:3000/) URL, in the case where someone already has the application running but doesn't have it open in the browser (I do this often).

Reply:
Great idea, I will do that!